### ISSUE TITLE
unittests fail on Win7

### ISSUE THREAD

**Issue by Zapunidi** [davidjamesca/ctypesgen#79]
I am running Python 3.6 32bit under Windows 7 and mingw compiler. This is well within requirements wiki page, saying that requirements are "Python, at least version 2.4"
Having changed tox ini file to python3.6 I encountered errors:
```
ctypesgen/test/testsuite.py::StdlibTest::test_getenv_returns_null FAILED [  2%]

================================== FAILURES ===================================
_____________________ StdlibTest.test_getenv_returns_null _____________________

self = <testsuite.StdlibTest testMethod=test_getenv_returns_null>

    def test_getenv_returns_null(self):
        """Related to issue 8. Test getenv of unset variable.
        """
        module = self.module
        env_var_name = "NOT SET"
        expect_result = None
        try:
            # ensure variable is not set, ignoring not set errors
            del os.environ[env_var_name]
        except KeyError:
            pass
>       result = module.getenv(env_var_name)
E       AttributeError: module 'temp' has no attribute 'getenv'

env_var_name = 'NOT SET'
expect_result = None
module     = <module 'temp' from '.\\temp.py'>
self       = <testsuite.StdlibTest testMethod=test_getenv_returns_null>

ctypesgen\test\testsuite.py:130: AttributeError

...

ERROR    ctypesgen:messages.py:38 Function "_atoi64" depends on an unknown typedef "__int64". Function "_atoi64" will not be output
ERROR    ctypesgen:messages.py:38 Function "_i64toa" depends on an unknown typedef "__int64". Function "_i64toa" will not be output
ERROR    ctypesgen:messages.py:38 Function "_wtoi64" depends on an unknown typedef "__int64". Function "_wtoi64" will not be output
ERROR    ctypesgen:messages.py:38 Function "_i64tow" depends on an unknown typedef "__int64". Function "_i64tow" will not be output
```
There may be a Windows incompatibility, though manual is misleading on the limits of Windows support. I had problems with processing mingw <sys/time.h> include and now I don't understand is it poor Windows support or what, having unittests fail.

**Comment by olsonse** [comment]
This was a bit weird testing on mingw.  The unit tests seemed to somewhat randomly fail, until I fixed two issues that didn't seem to be random.  Now the testsuite passes for me every time.

Commits 27a749861e95 and 7006cb27a8c0 fix unit tests at least on W10 with mingw-w64-x86_64.  Didn't try 32 bit, but it should work.

There is a good chance that issue #77 will continue to cause problems here until it is resolved (though for my Python 3.8.2 on mingw, it does not seem to show the same problems as #77).

**Comment by Zapunidi** [comment]
Consider reopening because MainTest tests will not run on Windows with python3.6. They call "python3 {args}" while on Windows python is called as just "python {args}". Just for test I removed "3" and all tests pass.

- pyexec = "python{}".format(sys.version_info.major)
+ pyexec = "python"

Windows python aliases were added in Python3.7 as far as I understood from this link: https://devblogs.microsoft.com/python/python-in-the-windows-10-may-2019-update/
